### PR TITLE
Update dxvk.conf

### PR DESCRIPTION
**Pull request by VincentProuchet**
Added hud config valuesalong with their definition the goal is to eliminate the necesity to rely on project readme for configuration

### CODE REVIEW

File: dxvk.conf [hunk 1]
```diff
@@ -339,6 +339,24 @@
 # environment variable is not set, otherwise it will be
 # ignored. The syntax is identical.
```
Comment (Blisto91) [inline dxvk.conf:340]: > The syntax is identical.

This still hints you to look at the readme, so i'd just note here that the values are comma separated.

File: dxvk.conf [hunk 2]
```diff
@@ -339,6 +339,24 @@
...
 
+# a valid value is : devinfo,fps,frametimes,pipelines,memory,version,api,compiler,
```
Comment (Blisto91) [inline dxvk.conf:342]: If the above feedback is addressed then this is probably redundant and can be left out.

File: dxvk.conf [hunk 3]
```diff
@@ -339,6 +339,24 @@
...
+
+#  here the different elements you can use
```
Comment (Blisto91) [inline dxvk.conf:344]: Capitalize the first letter

To make it consistent with the readme i'd probably change the wording to "options"
e.g "The following options are supported" or similar

File: dxvk.conf [hunk 4]
```diff
@@ -339,6 +339,24 @@
...
+#    pipelines	: Shows the total number of graphics and compute pipelines.
+#    descriptors: Shows the number of descriptor pools and descriptor sets.
+#    memory		: Shows the amount of device memory allocated and used.
+#    gpuload	: Shows estimated GPU load. May be inaccurate.
+#    version	: Shows DXVK version.
+#    api		: Shows the D3D feature level used by the application.
+#    cs			: Shows worker thread statistics.
+#    compiler	: Shows shader compiler activity
+#    samplers	: Shows the current number of sampler pairs used [D3D9 Only]
+#    scale=x	: Scales the HUD by a factor of x (e.g. 1.5)
```
Comment (Blisto91) [inline dxvk.conf:359]:
```suggestion
# - devinfo: Displays the name of the GPU and the driver version.
# - fps: Shows the current frame rate.
# - frametimes: Shows a frame time graph.
# - submissions: Shows the number of command buffers submitted per frame.
# - drawcalls: Shows the number of draw calls and render passes per frame.
# - pipelines: Shows the total number of graphics and compute pipelines.
# - descriptors: Shows the number of descriptor pools and descriptor sets.
# - memory: Shows the amount of device memory allocated and used.
# - gpuload: Shows estimated GPU load. May be inaccurate.
# - version: Shows DXVK version.
# - api: Shows the D3D feature level used by the application.
# - cs: Shows worker thread statistics.
# - compiler: Shows shader compiler activity
# - samplers: Shows the current number of sampler pairs used [D3D9 Only]
# - scale=x: Scales the HUD by a factor of x (e.g. 1.5)
```

File: dxvk.conf [hunk 5]
```diff
@@ -339,6 +339,24 @@
...
 # dxvk.hud =
```
Comment (Blisto91) [inline dxvk.conf:360]: Add a empty line before this

**Review by Blisto91 — COMMENTED** [review]
Hello and thank you for the PR.
I've noted some styling nits

**Comment by Blisto91** [comment]
Also try to make the commit name a bit informing so it can be seen just from your name what the commit does.

The common style would be something like
`[meta] Add dxvk.hud options to example config`

File: dxvk.conf (hunk 1 above)
Comment (VincentProuchet) [reply dxvk.conf:340]: I just wanted to avoid opening the config file and the github readme
I'm lazy

File: dxvk.conf (hunk 2 above)
Comment (VincentProuchet) [reply dxvk.conf:342]: A sample to guide out low level like me

File: dxvk.conf (hunk 4 above)
Comment (VincentProuchet) [reply dxvk.conf:359]: I will integrate the presentation to the next commit 
do you want another pull request ? with your requested modification ?

**Comment by VincentProuchet** [comment]
thanks for the code reviews and all the hints

**Pull request closed without merging**